src/record/export/usecases/__tests__/get/fixtures/can_download_files_including_special_characters.ts

```diff
@@ -0,0 +1,106 @@
+import type { KintoneRecordForResponse } from "../../../../../../kintone/types";
+import type { LocalRecord } from "../../../../types/record";
+import type { RecordSchema } from "../../../../types/schema";
+import path from "path";
+
+const fileInfo = {
+  contentType: "text/plain",
+  fileKey: "test-file-key",
+  name: ':"t/e\\|s?*t<>.txt',
+  size: "123456",
+};
+
+export const input: KintoneRecordForResponse[] = [
```

Comment:
IMO: I recommend you to add Attachment field in Table to your fixture.

Reply:
Fixed with 232d881fcbf5b7474965ce29d9752b38d859c6a3

Reply:
Fixed with 2102c7434d0c101b98707844840f3079322add3c